richtextfx/src/main/java/org/fxmisc/richtext/model/UndoActions.java

```diff
@@ -21,8 +20,8 @@
     default void redo() { getUndoManager().redo(); }
 
     default boolean isUndoAvailable() { return getUndoManager().isUndoAvailable(); }
-    default ObservableBooleanValue undoAvailableProperty() { return getUndoManager().undoAvailableProperty(); }
+    default Val<Boolean> undoAvailableProperty() { return getUndoManager().undoAvailableProperty(); }
 
     default boolean isRedoAvailable() { return getUndoManager().isRedoAvailable(); }
-    default ObservableBooleanValue redoAvailableProperty() { return getUndoManager().redoAvailableProperty(); }
+    default Val<Boolean> redoAvailableProperty() { return getUndoManager().redoAvailableProperty(); }
```

Comment:
You should have used a different garbage collector ;-) (and is that some reference to Scala's way of generics?)

> `getUndoManager()` should return `UndoManager<?>`

Yeah, I tried that and... wait, no I didn't try that :sweat_smile: